.github/workflows/milestone-automation.yml

```diff
@@ -0,0 +1,15 @@
+name: milestone-automation
+
+on:
+  schedule:
+    - cron: '0 * * * 6'
+  workflow_dispatch:
+
+jobs:
+  milestones:
+    uses: ConduitIO/automation/.github/workflows/milestones.yml@main
```

Comment:
Did you mean [`roadmap-milestones.yml`](https://github.com/ConduitIO/automation/blob/main/.github/workflows/roadmap-milestones.yml)?

Reply:
@lovromazgon ugh. Good catch. I can't believe I did that.